package.json

```diff
@@ -150,6 +152,13 @@
 		"react-dom": "16.8.6"
 	},
 	"devDependencies": {
+		"@automattic/calypso-build": "1.0.0",
```

Comment:
Some of these devDependencies got prolly added here accidentally since they're in `dependencies` as well.

Reply:
Fixed!